mne/channels/channels.py

```diff
@@ -1335,3 +1335,63 @@ def _get_ch_info(info):
     return (has_vv_mag, has_vv_grad, is_old_vv, has_4D_mag, ctf_other_types,
             has_CTF_grad, n_kit_grads, has_any_meg, has_eeg_coils,
             has_eeg_coils_and_meg, has_eeg_coils_only)
+
+
+def create_1020_montage_mapping(ch_names, midline="z12"):
+    r"""Return dict mapping from ROI names to lists of picks for 10/20 setups.
+
+    This passes through all channel names, and uses a simple heuristic to
+    separate channel names into three Regions of Interest: Left, Midline and
+    Right. The heuristic is that channels ending on any of the characters in
+    `midline` are filed under that heading, otherwise those ending in uneven
+    numbers to the left, those in even numbers to the right. Other channels
+    are ignored. This is appropriate for 10/20 files, but not for other channel
+    naming conventions.
+    If an info object is provided, lists are sorted from posterior to anterior.
+
+    Parameters
+    ----------
+    ch_names : instance of info | list
+        Object a list of channel names will be extracted from. The picks will
+        be in relation to the position in this list. If an info object, lists
+        will be sorted by y value position of the channel locations, i.e.,
+        from front to back.
+    midline : str
+        Names ending in any of these characters are stored under the `midline`
+        key. Defaults to 'z12'. Note that capitalization is ignored.
```

Comment:
You would assign C1/F1/C2/F2/... to the midline? I'd always assigned them to either left or right hemispheres.

Reply:
I'd prefer to have groups at least roughly even in size. You disagree here? We should of course follow conventions in our defaults.

Reply:
I thought the 10/20 convention was that only electrodes ending with z are assigned to the midline (but I don't have a reference for this claim). If your application requires even group sizes, then this behavior should be mentioned.

Reply:
The application would be plots like https://mne-tools.github.io/dev/_images/sphx_glr_plot_stats_cluster_erp_002.png ...

I'd worry about convention here most. If you're confident it's more usual, then I will change the default.

Reply:
A third opinion would be helpful, but I'm pretty confident that z electrodes define the midline. The question is do we want electrodes that are close to the midline (such as C1/C2 etc.) to be assigned to our midline category?

Reply:
Yes, indeed. If you think it's confusing, I'll change.